### PR TITLE
Added nationality, removed birth city and state

### LINKED ISSUE #737: Sign-up: change "where are you from" to country of birth & nationality
We will ask users for country of birth and nationality, instead of the full country, region and city of birth. This fits better with some future use cases for this profile data. 

image

The text and style for the tooltip is below: 

image

# Acceptance Criteria

- [ ] update profile model to include nationality (and remove birth_region and birth_city) (#775)
- [ ] update serialization for elastic search (#775)
- [ ] update profile API (#775); nationality is an optional field
- [ ] update react component
- [ ] use a simple country list for nationalities
- [ ] implement tool tip on click, consistent with other tool tips UX and styling

# Out of scope

- updating search results

### LINKED ISSUE #775: Update user profile models and API to accommodate nationality
nationality is optional 

We will facet on it in the elasticsearch index. This story includes updating the indexing code. 

part of #737

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #737 
Fixes #775 

#### What's this PR do?
Removes birth city and birth state or territory, and adds a nationality field. Note that we are using a list of countries as nationalities to choose from and not using the adjectival form discussed in the issue.

#### How should this be manually tested?
Run the migrations then go to the dashboard. You should be redirected to the profile with a validation error for the empty nationality field. Choose a country, then save. The validation error should go away and the nationality should show up when you view your own profile.

### CODE REVIEW

**Comment by gsidebo** [comment]
did you make changes to the user generation code to create this new `profiles/management/realistic_user_data.json` file? if so, can you push the changes after this merges?

**Comment by noisecapella** [comment]
No, I just edited it manually

File: static/js/components/PersonalForm.js [hunk 1]
```diff
@@ -105,28 +109,31 @@ export default class PersonalForm extends ProfileFormFields {
         </Cell>
         <Cell col={12}>
           <div className="section-header">
-            Where are you from?
+            Where are you from? <span
```
Comment (gsidebo) [inline static/js/components/PersonalForm.js]: i think it would be better to open the `<span` on a new line

Comment (noisecapella) [inline static/js/components/PersonalForm.js]: It needs to be on the same line to have React include the space between the `?` and the `<`. If they are on separate lines React will assume there is no space between them

Comment (gsidebo) [inline static/js/components/PersonalForm.js]: ok, fair enough

**Comment by gsidebo** [comment]
looks good 👍

**Pull request merged**